docs/understanding-airbyte/typing-deduping.md

```diff
@@ -30,39 +52,39 @@ Consider the following [source schema](https://docs.airbyte.com/integrations/sou
+| -------------------------------------------- | ---------------- | ----------------------------------------------------------------------------------------- | -------------------- | ---------------------- |
+| Successful typing and de-duping ⟶            | xxx-xxx-xxx      | { id: 1, first_name: “sarah”, age: 39, address: { city: “San Francisco”, zip: “94131” } } | 2022-01-01 12:00:001 | 2022-01-01 12:00:00﻿   |
+| Failed typing that didn’t break other rows ⟶ | yyy-yyy-yyy      | { id: 2, first_name: “evan”, age: “fish”, address: { city: “Menlo Park”, zip: “94002” } } | 2022-01-01 12:00:001 | 2022-01-01 12:00:00﻿   |
+| Not-yet-typed ⟶                              | zzz-zzz-zzz      | { id: 3, first_name: “edward”, age: 35, address: { city: “Sunnyvale”, zip: “94003” } }    | NULL                 | 2022-01-01 13:00:00﻿   |
 
 You also now see the following changes in Airbyte-provided columns:
 
 ![Airbyte Destinations V2 Column Changes](../release_notes/assets/destinations-v2-column-changes.png)
 
 ## Participating in Early Access
 
-You can start using Destinations V2 for BigQuery in early access by following the below instructions: 
+You can start using Destinations V2 for BigQuery in early access by following the below instructions:
 
 1. **Upgrade your BigQuery Destination**: If you are using Airbyte Open Source, update your BigQuery destination version to the latest version. If you are a Cloud customer, this step will already be completed on your behalf.
-2. **Enabling Destinations V2**: Create a new BigQuery destination, and enable the Destinations V2 option under `Advanced` settings. You will need your BigQuery credentials for this step. For this early release, we ask that you enable Destinations V2 on a new BigQuery destination using new connections. When Destinations V2 is fully available, there will be additional migration paths for upgrading your destination without resetting any of your existing connections. 
-    1. If your previous BigQuery destination is using “GCS Staging”, you can reuse the same staging bucket. 
-    2. Do not enable Destinations V2 on your previous / existing BigQuery destination during early release. It will cause your existing connections to fail.
+2. **Enabling Destinations V2**: Create a new BigQuery destination, and enable the Destinations V2 option under `Advanced` settings. You will need your BigQuery credentials for this step. For this early release, we ask that you enable Destinations V2 on a new BigQuery destination using new connections. When Destinations V2 is fully available, there will be additional migration paths for upgrading your destination without resetting any of your existing connections.
+   1. If your previous BigQuery destination is using “GCS Staging”, you can reuse the same staging bucket.
+   2. Do not enable Destinations V2 on your previous / existing BigQuery destination during early release. It will cause your existing connections to fail.
```

Comment:
```suggestion
   2. Do not enable Destinations V2 on your previous / existing destinations during early release. It will cause your existing connections to fail.
```